### ISSUE TITLE
Feature: add "Dark Mode"

### ISSUE THREAD

**Issue by folletto** [folletto/Bliphome#5]
Add a "Dark Mode".

Feature request by Daniel Rey López.

**Comment by folletto** [comment]
Via Daniel, the "prefers-color-scheme media query": https://bugs.chromium.org/p/chromium/issues/detail?id=889087 
It was apparently started three days ago.

**Comment by iam-robin** [comment]
I just came across the extension and it is exactly what I was looking for!
I would like to help to develop the extension further and I will focus on this issue first.

There will be a PR with my solution for this issue in a few days 🙂

**Comment by iam-robin** [comment]
just wanted to push my feature branch but I don't have the needed permissions:
"ERROR: Permission to folletto/Bliphome.git denied to iam-robin."

**Comment by folletto** [comment]
I've not done it myself, but I think you have first to [fork](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo) and the submit a [new PR](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) here once your branch is ready.
